### PR TITLE
Edit and delete comment

### PR DESCRIPTION
**Pull request by limkimhuor**

### CODE REVIEW

File: app/controllers/comments_controller.rb [hunk 1]
```diff
@@ -11,7 +12,30 @@ def create
     redirect_to request.referer || root_url
   end
 
+  def edit
+  end
+
+  def update
+    if @comment.update_attributes(comment_params)
```
Comment (RathanakSreang) [inline app/controllers/comments_controller.rb]: remove ()

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -11,7 +12,30 @@ def create
...
+      flash[:notice] = t "flashs.success"
+      redirect_to @comment.review
+    else
+      render 'edit'
```
Comment (RathanakSreang) [inline app/controllers/comments_controller.rb]: `"`

File: app/controllers/reviews_controller.rb [hunk 3]
```diff
@@ -1,8 +1,8 @@
 class ReviewsController < ApplicationController
   before_action :authenticate_user!, only: [:create, :destroy]
+  before_action :set_review, only: [:show, :edit, :update, :destroy]
```
Comment (RathanakSreang) [inline app/controllers/reviews_controller.rb]: `before_action :set_review, except: [:new, :create]`

**Comment by RathanakSreang** [comment]
Ok

File: app/controllers/comments_controller.rb [hunk 4]
```diff
@@ -11,7 +12,30 @@ def create
...
+    @comment.destroy
+    flash[:alert] = t "flashs.comment_delete"
+    redirect_to request.referer || root_url
+  end
+
   private
+  def set_comment
+    @comment = Comment.find params[:id]
+  end
   def comment_params
```
Comment (HorNorin) [inline app/controllers/comments_controller.rb]: insert one newline

**Comment by HorNorin** [comment]
ok

**Pull request merged**